File: PRAKTIKUM MODUL 7/SOAL 3/PRAK603-2210817210008-Bima Sanjaya.c
#include <stdio.h>
#include <stdlib.h>
int main(){
    int x,y;
    scanf("%d %d", &x, &y);
    if(x!=y){
        printf("Jumlah tidak sama");
        exit(1);
    }
    int z[x*y];
    for(int i=1;i<=(x*2);i++){
        scanf("%d", &z[i]);
    }
    for(int i=1;i<=x;i++){
        printf("%d ", z[i]*z[i+x]);
    }
    return 0;
}

File: PRAKTIKUM MODUL 7/SOAL 3/PRAK603-2210817210008-Bima Sanjaya.py
a,b=map(int,input().split())
if (a==b):
    b1=list(map(int,input().split()))
    b2=list(map(int,input().split()))
    for i in range (a):
        print(b1[i]*b2[i],end=' ')
else:
    print("Jumlah tidak sama")